docs/source/configuration/overview.mdx

```diff
@@ -810,6 +810,16 @@ experimental_graphql_validation_mode: new
 
 This is an experimental option while we are still finding edge cases in the new implementation, but it will become the default in the future.
 
+### API Schema Generation Mode
```

Comment:
I documented this option but I don't think we should really expect users to use this, the visible impact of the change and the risk of something being wrong is *much* lower than it was with `experimental_graphql_validation_mode`. We could just do it silently?

Reply:
I think we should do it silently.